src/types/types.spec.ts

```diff
@@ -383,10 +390,14 @@ chains.forEach(({ name, constructor: NodeConstructor }) => {
               version.includes(node.clientVersion).should.be.true();
             });
           });
-          describe(`${name}.isRunning() while running`, function() {
-            it('should resolve with a boolean indicating that the node is running', async function() {
+          describe(`${name}.isRunning() while running local`, function() {
```

Comment:
It wasn't specific to Fantom, it was just I was missing some of the updated docker related methods since I started from master originally instead of dev. At the time it was helpful for me to separate the tests to troubleshoot what was going wrong as I couldn't see which of the two was failing

Reply:
All right. For now, please just revert that piece to what it was before and we can talk in the future about updating the tests.